### ISSUE TITLE
Explore Idea of Holding Raspberry Pi & Ant Via Clips Instead of Screws

### ISSUE THREAD

**Issue by keenanjohnson** [Ribbit-Network/ribbit-network-frog-sensor#138]
In #132 we discussed replacing the screws that hold the raspberry pi and the circular hole that holds the antennae with clips that are printed as part of the enclosure.

Not sure if that's practical or reliable, but an interesting idea to consider and try.

**Comment by eaudiffred** [comment]
Added a branch in OnShape off of the final V3 enclosure to start experimenting with the clips

**Comment by eaudiffred** [comment]
I have this prototype printing:
![image](https://user-images.githubusercontent.com/88562047/183709788-afe1d4e8-42e8-4880-8aef-18732d717460.png)
![image](https://user-images.githubusercontent.com/88562047/183709984-54fa1a50-936a-4f8c-8308-f7e11d68352f.png)

**Comment by eaudiffred** [comment]
The clips are defiantly small, but didn't break off when I manually flexed them.  I have an antenna on order, hopefully it gets to me today or tomorrow and I can use the board in my V1 sensor to test fit.  Hopefully I can get to that this evening.  I'll share some pictures of the results.  

Totally up to you if you want to do a test as well.  

I also made some small modifications to the lid and enclosure base to remove the two screws.  I think the lid snaps in pretty well, and two less screws is a good thing.  It also allowed me to remove some material from the base.  
![image](https://user-images.githubusercontent.com/88562047/183916836-dc53fa4d-e965-4a23-bd50-b51abecb477c.png)
![image](https://user-images.githubusercontent.com/88562047/183916914-11032aa5-8e4c-4c64-b09e-9104fbe3a132.png)

**Comment by eaudiffred** [comment]
The board clips didn't work.  I made the dimensions too exact.  Need to add in some clearance for the post height, clip height and clip width.

**Comment by keenanjohnson** [comment]
The new lip on the enclosure looks great. Thanks so much for tackling this!

**Comment by eaudiffred** [comment]
I received the antenna yesterday and tried it out.  The base clip was too tight and snapped.  I'll tweak both sets and try another test print.

**Comment by eaudiffred** [comment]
This isn't going as smoothly as I hoped.  I'll need some more time.  I think it would be best to launch V3 as is concerning the enclosure.

**Comment by keenanjohnson** [comment]
Makes sense as this is a pretty difficult challenge it seems to me! It's easy to do another revision after once we figure it out :)

**Comment by eaudiffred** [comment]
Here is my latest idea for the clip design.  
![image](https://user-images.githubusercontent.com/88562047/188213938-7a7dc04d-b6ab-4ee4-b7c6-457bfeeef15c.png)
In the previous versions the vertical/post style clips kept snapping.  In this version, only the zig-zag shape clips will flex.

**Comment by keenanjohnson** [comment]
Awesome! Do you need any help doing a test print @eaudiffred ?

**Comment by eaudiffred** [comment]
I can get a test printed this weekend.  But obviously feel free to print one as well!  
How is the new printer working out?  Big difference?

**Comment by keenanjohnson** [comment]
The new printer is much nicer. The auto bed leveling sensor saves a ton of time it seems and I've zero failed prints so far (only printed a few test things though) which is super nice.

**Comment by eaudiffred** [comment]
Much better results with the clips shows in the CAD model above.  Some tweaking to do on the upper antenna clip and tightening up the board clips.

**Comment by eaudiffred** [comment]
I'm going to pause on this until we determine if we need to move to V4 of the sensor to support new hardware.
